frontend/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.3"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
React does this really annoying thing where `react-scripts test` only runs tests on files that have changes since the last commit.
Can you add a script that runs all the tests regardless? Just add the following line
```
    "test-all": "react-scripts test --watchAll=false"
```

Reply:
added in 01db205
Typo fixed in b937456